src/freenet/support/CurrentTimeUTC.java

```diff
@@ -29,30 +40,45 @@ public static Date get() {
 	}
 
 	/**
-	 * Get the current time in milliseconds. 
-	 * 
-	 * In the current implementation, this just returns System.currentTimeMilis(). You should however use CurrenTimeUTC.getInMillis() instead because
-	 * the JavaDoc of System.currentTimeMilis() does not explicitly state what time zone it returns. Therefore, by using this wrapper function, your code
-	 * clearly states that it uses UTC time. 
+	 * Get the current time in milliseconds since epoch.
+	 * @return the difference, measured in milliseconds, between the current time and midnight, January 1, 1970 UTC.
+	 * @deprecated use {@link System#currentTimeMillis()}
 	 */
+	@Deprecated
 	public static long getInMillis() {
 		return System.currentTimeMillis();
 	}
 
+	/**
+	 * Get the current calendar year.
+	 * @deprecated use {@link java.time.LocalDate#now(java.time.ZoneId) LocalDate.now(ZoneOffset.UTC)}
+	 */
+	@Deprecated
 	public static int getYear() {
 		synchronized(mCalendar) {
 			mCalendar.setTimeInMillis(System.currentTimeMillis());
 			return mCalendar.get(Calendar.YEAR);
 		}
 	}
 
+	/**
+	 * Get the zero-indexed current calendar month.
+	 * @return the zero-indexed month number, where 0 indicates January and 11 indicates December
+	 * @deprecated use {@link java.time.LocalDate#now(java.time.ZoneId) LocalDate.now(ZoneOffset.UTC)}
```

Comment:
this replacement does not return the month. It also does not explain the sad peculiarities of the month indexing.